save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/TopBar.kt

```diff
@@ -109,10 +117,28 @@ fun topBar() = FC<TopBarProps> { props ->
                     .split(URL_PATH_DELIMITER)
                     .filterNot { it.isBlank() }
                     .apply {
-                        foldIndexed("#") { index: Int, acc: String, pathPart: String ->
-
-                            val currentLink = "$acc/$pathPart"
-
+                        val insideTab = location.pathname.substringBeforeLast("?").let {
+                            if (it.contains(OrganizationMenuBar.regexForUrlClassification)) {
+                                "organization"
```

Comment:
Can these constants be extraced into corresponding `TabMenuBar`s?

Reply:
Done